### DISCUSSION TITLE
future of nexus

### DISCUSSION THREAD

**Discussion by Manubi** [graphql-nexus/nexus-prisma#228] [General]
Hi,
as the last update of the package was in december last year and even a bump to prisma v4 seems to not have any priority do you think there will be a future of nexus?

**Comment by futurebenmorris** [comment]
I second this. Would be great to get an update on this. Things have fizzled out development wise. Prisma set out lofty goals and a detailed plan for this, so keen to know if it's still going to get some investment.

**Comment by orjellingsen** [comment]
Sadly, we can not expect this project to be supported anymore. 

It would be great if this repo was updated with information about the current state. At the moment this information is only found on [twitter](https://mobile.twitter.com/nikolasburk/status/1554381468998291456) or inside a [150 comment long thread](https://github.com/graphql-nexus/nexus-plugin-prisma/issues/1039) on another repo (as far as I know).

Having followed the discussion over at nexus-plugin-prisma for a while, it seems like the general conclusion is to use Pothos instead of Nexus from now on. 

Lucky for us, Pothos seems like an amazing project. They even have a pretty great prisma plugin :)

https://pothos-graphql.dev/

**Reply by futurebenmorris** [reply]
Agreed, I've got an early stage product working with Nexus-Prisma, keen to know if I should migrate to Pothos or an alternative.

**Reply by villesau** [reply]
Any guides for migration?

**Comment by rostislav-simonik** [comment]
I'm taking over the nexus-prisma maintenance, more info here https://github.com/graphql-nexus/nexus-prisma/issues/279. I'll also prepare a roadmap soon, which I'll try to keep up to date.

**Reply by henryson** [reply]
@rostislav-simonik what is the status of nexus-prisma now? I am about to migrate from paljs to nexus-prisma, but looking around it seems this project is getting less attention? Do you think it is a good idea for me to switch to nexus-prisma, or should I look for something else?

**Reply by Manubi** [reply]
Have a look at drizzle. Probably the better choice.

**Reply by henryson** [reply]
I had a quick look at drizzle, but it seems to replace Prisma? I want to keep Prisma but switch "middleware" from paljs to something different. Do you mean that drizzle could help with this also? @Manubi

**Reply by Manubi** [reply]
Have a look at

https://orm.drizzle.team/docs/graphql

If i remember correctly there is a tool that does the same. Maybe check out drizzles discord server and ask there or in their github discussions.
